### ISSUE TITLE
Manpage: Fix Inconsistencies

### ISSUE THREAD

**Issue by tomschr** [openSUSE/docmanager#36]
After installing docmanager, Frank reviewed the manpage and found these issues:
## `set` Subcommand
- Definition of "property" is missing
- Definition of "standard properties" is missing
- Options `--maintainer`, `--status`, `--deadline`, `--priority`, `--translation`, and `--languages` should contain the allowed datatype
- The sentence "Note: This option overwrites -p and other ..." is a bit misleading. What about this:
  
  ```
  Note: If you use -p maintainer FOO, --maintainer has a higher priority than -p
  ```
## `get` Subcommand
- Shouldn't be the standard properties as options also available like in `set`?
## `del` Subcommand
- Shouldn't be the standard properties as options also available like in `set`?

**Comment by mschnitzer** [comment]
@tomschr get & del: No there shouldn't be the standard properties like in set. This has been discussed 2-3 weeks ago with Frank. Frank said: It's not really needed.

**Comment by mschnitzer** [comment]
I discussed it again with @fsundermeyer and he says that we can implement these options for `set` in 3.1. It is just a "feature" and not really needed in 3.0.

**Comment by mschnitzer** [comment]
@tomschr In commit https://github.com/openSUSE/docmanager/commit/c99ca9deac987b543b4ca2f71978d9e89db86310 I've added your suggestion.

**Comment by mschnitzer** [comment]
@tomschr I will close it since I fixed your/franks suggestions in c99ca9d.
